omero/developers/index.txt

```diff
@@ -115,6 +115,7 @@ Web
     Web/WritingViews
     Web/WritingTemplates
     Web/CSRF
+    Web/Webclient
```

Comment:
I'd suggest this would be better placed as the third item from the top on this list, it certainly doesn't make any sense to me to put it last.